packages/IMAPClient-Tests.package/ICDeployment.class/README.md

```diff
@@ -1 +1 @@
-This class supportes the automatic release via travis.
\ No newline at end of file
+This class supportes the automatic release via travis. It is not used for teting but enables automatic sar bulding.
```

Comment:
This class *supports automatic releases* via Travis.
It is not used for *testing*.